### ISSUE TITLE
Warehouse losing items

### ISSUE THREAD

**Issue by schradieck** [P3pp3rF1y/AncientWarfare2#417]
i run a small faction based server for me and my brothers, and one of my brothers is using ancient warfare 2 and we have noticed that warehouse is loosing a lot of it's inventory every time the server gets restarted

**Comment by Dark-Matter-001** [comment]
After using the latest version (2.4.115 at this time) I too have been noticing items going missing from my warehouse.  I think it might be an issue with the warehouse not correctly keeping track of what storage blocks it is using, storing items in space that doesn't exits, and then after a server restart those items are gone.

Here's an example of how bugged my warehouse setup got after 3-4 days and 5-7 server restarts:
https://imgur.com/a/vYpOq

So it seems like something breaks and stops the controllers from updating what storage blocks it has and what's in them?  After that items get stored in non existent space and eventually go missing.

Would backup mods that disable saving while they zip the world folder cause problems as well? (using FTB Utilities 1.7.10-1.0.17.6)

The only other thing I can think of, that I just noticed and may contribute to breaking the warehouse, is that my controllers are in chunk loaded chunks while the the storage blocks on the very back wall are not.  So when the server restarts I assume the controllers and maybe the nearest storage blocks would get loaded but the back wall doesn't.  Would that break it in the ways I've experienced or cause items to randomly vanish over time?
